spec/struct.dd

```diff
@@ -181,23 +189,12 @@ static S q = { 1, 2 }; // q.a = 1, q.b = 2, q.c = 0, q.d = 7
 ------
 
         $(P Struct literals can also be used to initialize statics, but
-        they must be evaluable at compile time.)
+        they must be evaluatable at compile time.)
```

Comment:
And the other instances in the context lines here. These lines are being changed anyway.